lib/node/node.ts

```diff
@@ -83,3 +85,28 @@ if (httpClient && httpClient.ClientRequest) {
     }
   };
 }
+
+function patchNextTick() {
+  let setNative = null;
```

Comment:
It seems weird to use this local variable instead of moving `scheduleTask` inside of `patchMethod`, but I guess you're trying to limit nested closures?